development-defaults.ini

```diff
@@ -151,10 +151,6 @@ no_shirt     = 0
 
 [enums]
 
```

Comment:
No reason to make these part of the core repo; we can add them in ``development.ini`` if a particular event needs that.